catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopCatalogPropertiesMetadata.java

```diff
@@ -41,7 +41,7 @@ public class HadoopCatalogPropertiesMetadata extends BaseCatalogPropertiesMetada
               PropertyEntry.stringOptionalPropertyEntry(
                   LOCATION,
                   "The storage location managed by Hadoop fileset catalog",
-                  true /* immutable */,
+                  false /* immutable */,
```

Comment:
Do you test if it works or not if user changes this property?

Reply:
I think you should have a IT to cover this.

Reply:
added